packages/common-all/src/types/configs/commands/commands.ts

```diff
@@ -36,7 +34,6 @@ export function genDefaultCommandConfig(): DendronCommandConfig {
   return {
     lookup: genDefaultLookupConfig(),
     randomNote: genDefaultRandomNoteConfig(),
-    insertNote: genDefaultInsertNoteConfig(),
```

Comment:
reminder of a previous discussion we had about moving `workspace.templateHierarchy` to the `commands.applyTemplate` namespace since this is only affects the command.

Reply:
yep, merged :)

https://github.com/dendronhq/dendron/pull/3069